运行报错:[ERROR] uvicorn | Application startup failed. Exiting.

今天初次安装然后输入hb run运行报错如下，python版本为python-3.9.6，切换过python-3.8.10也是一样的报错
初步怀疑是哪里依赖更新导致的报错，求解谢谢~

PS C:\Users\XXX\Desktop\Code\haruka-bot> hb run
07-23 21:31:07 [INFO] nonebot | NoneBot is initializing...
07-23 21:31:07 [INFO] nonebot | Current Env: prod
07-23 21:31:08 [INFO] nonebot | Succeeded to import "nonebot_plugin_apscheduler"
07-23 21:31:08 [WARNING] haruka_bot | 检测到当前为 Windows 系统，已自动注入猴子补丁
07-23 21:31:08 [INFO] nonebot | Succeeded to import "haruka_bot"
07-23 21:31:08 [INFO] nonebot | Running NoneBot...
07-23 21:31:08 [INFO] uvicorn | Started server process [12032]
07-23 21:31:08 [INFO] uvicorn | Waiting for application startup.
07-23 21:31:08 [INFO] nonebot_plugin_apscheduler | Scheduler Started
07-23 21:31:08 [INFO] haruka_bot | 正在检查 Chromium 更新
07-23 21:31:08 [ERROR] uvicorn | Traceback (most recent call last):
  File "c:\users\xxx\appdata\local\programs\python\python39\lib\site-packages\starlette\routing.py", line 526, in lifespan
    async for item in self.lifespan_context(app):
  File "c:\users\xxx\appdata\local\programs\python\python39\lib\site-packages\starlette\routing.py", line 467, in default_lifespan
    await self.startup()
  File "c:\users\xxx\appdata\local\programs\python\python39\lib\site-packages\starlette\routing.py", line 504, in startup
    handler()
  File "c:\users\xxx\appdata\local\programs\python\python39\lib\site-packages\haruka_bot\utils\browser.py", line 59, in install
    main()
  File "c:\users\xxx\appdata\local\programs\python\python39\lib\site-packages\playwright\__main__.py", line 27, in main
    sys.exit(completed_process.returncode)
SystemExit: 0
07-23 21:31:08 [ERROR] uvicorn | Application startup failed. Exiting.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -30,7 +30,7 @@ nonebot-adapter-cqhttp = "^2.0.0-alpha.13"
 nonebot2 = "^2.0.0a13.post1"
 pillow = "^8.2.0"
 tortoise-orm = {extras = ["asyncpg"], version = "^0.17.3"}
-playwright = "^1.11.2"
+playwright = "^1.13.0"
 appdirs = "^1.4.4"
 
 [tool.poetry.dev-dependencies]

--- a/src/plugins/haruka_bot/utils/browser.py
+++ b/src/plugins/haruka_bot/utils/browser.py
@@ -56,7 +56,10 @@ def install():
     import sys
     from playwright.__main__ import main
     sys.argv = ['', 'install', 'chromium']
-    main()
+    try:
+        main()
+    except SystemExit:
+        pass
 
 
 def delete_pyppeteer():

